spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'add some comments' do
+    author = User.new(name: 'Tom', photo: 'Tom.png', bio: 'bio', posts_counter: 0)
+    post = Post.new(title: 'New post', text: 'Good evening', author:, like_counter: 0, comments_counter: 0)
+    post.save!
```

Comment:
- [ ] This test was not passing and showing a syntax error on line `8` where you have `author:,`. Kindly fix this by renaming the author variable so that you can add it as the author value. See suggestion below, which will make the test pass and linter will not complain.

```suggestion
    user = User.new(name: 'Tom', photo: 'Tom.png', bio: 'bio', posts_counter: 0)
    post = Post.new(title: 'New post', text: 'Good evening', author: user, like_counter: 0, comments_counter: 0)
    post.save!
```

Reply:
@julie-ify How can i write tests for update_comments_counter, update_posts_counter, update_likes_counter because these methods are private.